tp1/tp1.go

```diff
@@ -23,18 +43,105 @@ type Carrito struct {
 // CalcularPrecios recibe un arreglo de los IDs de productos y calcula,
 // para cada super mercado, cuanto saldria comprar esos productos ahi.
 // Retorna un slice de carritos, donde se tiene uno para cada super mercado.
-func (p Productos) CalcularPrecios(ids ...int) []Carrito {
-	return nil
+func (p Productos) CalcularPrecios(ids []int) []Carrito {
+
+	chango := map[string]int{}
+
+	var id string
+
+	for i := 0; i < len(p); i++ {
+
+		for _, valor := range ids {
```

Comment:
Podriamos poner los ids en un mapa y ahorrarnos hacer un for por cada producto en `p`. Iterariamos `ids` una sola vez y despues se harian lookups O(1) por cada producto en `p`